aten/src/ATen/cpu/vec256/vec256_float_neon.h

```diff
@@ -282,6 +282,19 @@ template <> class Vec256<float> {
     }
     return mask;
   }
+  Vec256<float> isnan() const {
+    __at_align32__ float tmp[size()];
+    __at_align32__ float res[size()];
+    store(tmp);
+    for (int64_t i = 0; i < size(); i++) {
```

Comment:
Why `int64_t`, when `decltype(size())` is int?

Reply:
int should be enough.